wp-content/themes/goonj-crm/functions.php

```diff
@@ -419,8 +419,67 @@ function goonj_is_volunteer_inducted( $volunteer ) {
 function goonj_custom_message_placeholder() {
 	return '<div id="custom-message" class="ml-24"></div>';
 }
+
 add_shortcode( 'goonj_volunteer_message', 'goonj_custom_message_placeholder' );
 
+function generate_volunteer_button_html($buttonUrl) {
```

Comment:
I've give this feedback earlier as well – any function in the theme's function file needs to be prefixed by `goonj_`

Reply:
updated with prefix